src/styles/components/_presentation.scss

```diff
@@ -164,15 +160,15 @@
   text-overflow: ellipsis;
 }
 
-.presentation .thumbnail .caption .size {
+.presentation .caption .size {
```

Comment:
Still i think you should select on thumbnail as these styles are probably only valid for thumbnails.